src/test/validatorTest.ts

```diff
@@ -704,7 +705,7 @@ describe('validator', () => {
             let result = validator.validateFile(input);
             expect(result).to.have.deep.property('templateValid', false);
             expect(result['errors']['crit']).to.have.lengthOf(1);
-            expect(result['errors']['crit'][0]['message']).to.contain('Required property Key missing for type Tag');
+            expect(result['errors']['crit'][0]['message']).to.contain('Required property Key missing for type AWS::EC2::Instance.Tag');
```

Comment:
What about other resource tags on things like DynamoDB tables and Lambda functions?